public/apps/login/login-page.tsx

```diff
@@ -237,7 +237,10 @@ export function LoginPage(props: LoginPageDeps) {
             );
           }
 
-          if (authOpts.length > 1) {
+          if (
+            authOpts.length > 1 &&
+            (!authOpts.includes(AuthType.PROXY) || authOpts.length !== 2)
```

Comment:
Can you explain why this additional check is needed?

Reply:
The login page thinks that if there are multiple auth types we must have multiple buttons that need a horizontal separating bar. Since there is not a button for Proxy auth the bar is just separating blank space from the login button and looks odd.

Reply:
@stephen-crawford What happens if proxy is the only login method?